src/app/models/evt-models.ts

```diff
@@ -127,6 +127,23 @@ export interface WitnessGroup {
     groupId: string;
 }
 
+export type ApparatusEntriesList = Map<ApparatusEntry>;
+
+export interface ApparatusEntry extends GenericElementData {
+    id: string;
+    content: ApparatusReading[];
+    note: NoteData;
+    variance: number;
+    originalEncoding: string;
+}
+
+export interface ApparatusReading extends GenericElementData {
+    tagname: string;
```

Comment:
Why do you need to store tag name information?

Reply:
I need it to distinguish the lemma from other readings when they are displayed in the component. Do you think there is another way?

Reply:
Done